frontend/packages/pipelines-plugin/src/components/pipelineruns/__tests__/PipelineRunsResourceList.spec.tsx

```diff
@@ -28,6 +30,12 @@ describe('PipelineRunsResourceList:', () => {
     expect(wrapper.find(ListPage).props().badge).toBeNull();
   });
 
+  it('Should not render the badge in the list page if the pipeline GA operator is installed', () => {
+    jest.spyOn(operatorUtils, 'usePipelineOperatorVersion').mockReturnValue({ version: '1.4.0' });
```

Comment:
Heh, your other PR has a variable instead of the magic string. Minor, especially given this is a test, but FYI.